File: Latihan b/php/Product.php
<?php
// deklarasi kelas product
class Product{
	//variabel lokal
	private $price;
	private $idproduct;
	//konstruktor
	public function __construct(){
		$this->price = 0;
        $this->idproduct = "";
	}
	/*
	public function __construct($price = 0, $idproduct = ""){
		$this->price = $price;
		$this->idproduct = $idproduct;
	}
	*/
	//get dan set setiap variabel
	public function setprice($price){
		$this->price = $price;
	}
	public function getprice(){
		return $this->price;
	}
	public function setidproduct($idproduct){
		$this->idproduct = $idproduct;
	}
	public function getidproduct(){
		return $this->idproduct;
	}
	public function __destruct(){
	}
}
?>

File: Latihan a/java/product.java
// deklarasi kelas produk
class product{
    // variabel privat
    private int price;
    private int idproduct;
    // method public dan konstruktor
    public product(){

    }
    public product(int price, int idproduct){
        this.price=price;
        this.idproduct=idproduct;
    }
    // get dans set semua variabel
    public int getprice(){
        return price;
    }
    public void setprice(int price){
        this.price = price;
    }
    public int getidproduct(){
        return idproduct;
    }
    public void setidproduct(int idproduct){
        this.idproduct = idproduct;
    }
}